### PR TITLE
Update faq with npm integration information

### PR DESCRIPTION
**Pull request by VitaliyR**
Hi there.

Recently I've encountered a problem when I was need to override `main` attribute of some dependent  `package.json`.

Actually, I was need to grab only one .js file, but package `main` pointed to big compiled file.

After debugging for a while I've found, that brunch already supports overriding dependent packages manifests attributes not only for bower but for npm also.

I decided to update FAQ to help brunch's newcomers, maybe some of them facing it too.

### CODE REVIEW

**Comment by paulmillr** [comment]
Thanks.

File: docs/faq.md [hunk 1]
```diff
@@ -22,6 +22,56 @@ To add packages to your project:
 
 Example app with Bower integration: http://github.com/paulmillr/ostio
 
+## How to use NPM as client-side package manager?
+
+Brunch supports handling client-side dependencies using [NPM](https://npmjs.com) package manager.
+
+To do so, enable NPM integration in config. By default, brunch would look up for compatible node packages which can be used on client. You can tell branch to use explicit list of packages via `whitelist` attribute.
```
Comment (goshakkk) [inline docs/faq.md]: `whitelist` is gone in master (and so, will be gone in 2.2)

Comment (VitaliyR) [inline docs/faq.md]: I've read somewhere that 2.2 will predict which packages user is needed.
I found commits where info about whitelist & blacklist packages are removed. Too bad that we're updating docs before release - for instance, if I'm using latest published version I can't just to 'follow the docs'.

Should I remove from FAQ that information?

Comment (goshakkk) [inline docs/faq.md]: You should follow the docs at the tag not master?
On Thu, Jan 21, 2016 at 12:13 PM Vitaliy Ribachenko <
notifications@github.com> wrote:

> In docs/faq.md
> <https://github.com/brunch/brunch/pull/1120#discussion_r50375274>:
>
> > @@ -22,6 +22,56 @@ To add packages to your project:
> >
> >  Example app with Bower integration: http://github.com/paulmillr/ostio
> >
> > +## How to use NPM as client-side package manager?
> > +
> > +Brunch supports handling client-side dependencies using [NPM](https://npmjs.com) package manager.
> > +
> > +To do so, enable NPM integration in config. By default, brunch would look up for compatible node packages which can be used on client. You can tell branch to use explicit list of packages via `whitelist` attribute.
>
> I've read somewhere that 2.2 will predict which packages user is needed.
> I found commits where info about whitelist & blacklist packages are
> removed. Too bad that we're updating docs before release - for instance, if
> I'm using latest published version I can't just to 'follow the docs'.
>
> Should I remove from FAQ that information?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brunch/brunch/pull/1120/files#r50375274>.
>
-- 
Cheers,
Gosha

Comment (VitaliyR) [inline docs/faq.md]: Then we need to update ./README.md with the link to latest released docs..? I mean to the docs in latest tag. 

About this one, I will update PR with newest information.

**Comment by VitaliyR** [comment]
Moved to https://github.com/brunch/brunch/pull/1128

**Pull request merged**